theme_5/src/styles/vendor/product.scss

```diff
@@ -0,0 +1,132 @@
+
+// PRODUCT STYLES
+
+.product-section {
+  padding: 0 15px;
+}
+
+.product-box {
+  border: 1px solid $cerise-red;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.product-views {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  img {
+    margin-bottom: 30px;
+  }
+}
+
+.product-info {
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  position: relative;
+}
+
+.product-title {
+  color: $black-solid;
+  font-family: $playfair-font;
+  font-size: 24px;
+  font-weight: bold;
+  text-transform: capitalize;
+}
+
+.product-description {
+  color: $emperor;
+}
+
+.view-box {
+  height: 86px;
+  margin: 25px 0;
+  position: relative;
+  width: 86px;
+
+  .after {
+    color: rgb(232, 54, 113);
+    display: none;
+    height: 100%;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+  }
+
+  &:hover .after {
+    background: rgba(232, 54, 113, 0.6);
+    display: block;
+  }
+}
+
+.input-radio {
+  display: none;
+
+  &:checked + .label-color {
+    box-shadow: inset 0 0 10px $cerise-red;
+    opacity: 0.6;
+  }
+}
+
+.label-color {
+  border-radius: 100%;
+  height: 30px;
+  margin: 0 10px;
+  width: 30px;
+}
+
+.pink {
+  background: pink;
+}
+
+.red {
+  background: crimson;
```

Comment:
Color `crimson` should be written in hexadecimal form as `#dc143c`<br>Color literals like `crimson` should only be used in variable declarations; they should be referred to via variable everywhere else.